ftw/subsite/upgrades/20160816161334_make_subsite_linkable/tinymce.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<object>
+  <linkable>
+    <element value="ftw.subsite.Subsite" purge="False"/>
```

Comment:
Why does this look differently than `profiles/default/tinymce.xml`?
- no `<resourcetypes>`
- `purge` on `<element>` instead of `<linkable>`